src/coreclr/jit/importer.cpp

```diff
@@ -10343,18 +10343,34 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 // TODO: enable for X86 as well, it currently doesn't support memset/memcpy helpers
```

Comment:
x86 supports memset/memcpy helpers now (this can be a follow up change)

Reply:
Yes, it's part of a follow up PR because it involves removal of `GT_STORE_DYN_BLK` op, definitely didn't want to do it as part of this PR